pkg/helmcli/adder.go

```diff
@@ -0,0 +1,134 @@
+package helmcli
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
+	"github.com/gojekfarm/albatross/pkg/logger"
+
+	"github.com/gofrs/flock"
+	"gopkg.in/yaml.v2"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/repo"
+)
+
+const timeout time.Duration = 30 * time.Second
+
+type adder struct {
+	flags.AddFlags
```

Comment:
Do we need flags to be embedded here?

Reply:
We can think of an adder object being a first class action item, thus being self sufficient in what actions it's going to do.
The parameters for the action is part of the flag and hence embedded the flag.

since the struct is internal it won't be accessible by external users.